torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -304,6 +307,15 @@ def get_rref_debug_info():
 
 
 class RpcTest(RpcAgentTestFixture):
+    def _skip_if_tensorpipe_agent(old_func):  # noqa
+        def decorator(self):
+            return unittest.skipIf(
+                self.rpc_backend == rpc.backend_registry.BackendType.TENSORPIPE,
+                "This test is not yet supported in the Tensorpipe Agent"
+            )(old_func)
```

Comment:
Tip: `functools.wraps` is a nice tool to help make wrapper functions: it makes sure to also propagate some metadata from the wrapped function to the wrapper one (like docstrings, annotations, ...). I don't know if it matter here, but if you want to have a look: https://docs.python.org/3.8/library/functools.html#functools.wraps